extensions/spotify-controls/src/shufflePlay.ts

```diff
@@ -0,0 +1,32 @@
+import { runAppleScriptSilently } from "./utils";
+
+export default async () => {
+  const script = `
+
+    tell application "Spotify"
+
+      if not application "Spotify" is running then
+        
+        activate
+
+        repeat until application "Spotify" is running
+          delay 1
+        end repeat
+  
+      end if
+
+      set shuffling to true
+
+      set spotify_playing to (player state = playing)
+
+      if not spotify_playing then
+    
+        play
+        
+      end if
+
+    end tell
+  `;
```

Comment:
```suggestion
  const script = `
    tell application "Spotify"
      if not application "Spotify" is running then
        activate
        repeat until application "Spotify" is running
          delay 1
        end repeat
      end if
      set shuffling to true
      set spotify_playing to (player state = playing)
      if not spotify_playing then
        play
      end if
    end tell
  `;
```

Reply:
Good suggestion - I adopted that style.

This commit is out of date.